community/blog/2022-06 .NET 6 finally comes to Arcus Templates in v0.8 release/README.md

```diff
@@ -0,0 +1,112 @@
+# .NET 6 finally comes to Arcus Templates in v0.8 release, together with new templates and features
+The v0.8 release of Arcus Templates ends the journey of adding .NET support to Arcus. This release also contains tons of new features and useful changes. This run-down post will guide you through the biggest ones.
```

Comment:
When I read this first sentence, it's like this is the last release where we support .NET.
I would rewrite this to something more like 

> With the v0.8 release of Arcus Templates, we now fully support .NET 6.0 in Arcus.

Or something better :)

Reply:
> When I read this first sentence, it's like this is the last release where we support .NET.

🤔 that's a different take. Seems like 'X comes to Y' that it is finally there, not leaving 😄 .

Reply:
The proposed title doesn't include the other stuff, and since .NET 6 is only < 1/3 of the post, it doesn't seem right to only include that in the title.
`Arcus Templates v0.8 finally brings .NET 6 support to Arcus, together with new templates and features`. Something like this?

Reply:
Yes, that is ok.